>From 27a12db26aef77b7c9b1f7a10e134d6bda9a3397 Mon Sep 17 00:00:00 2001
From: Andreas Beckmann <anbe@debian.org>
Date: Fri, 13 Dec 2013 01:18:28 +0100
Subject: [PATCH] dh_installdeb: add support for d-m-h commands
 symlink_to_dir, dir_to_symlink

these two new dpkg-maintscript-helper commands were added in dpkg 1.17.2
(and fixed in 1.17.5, therefore using this version for Pre-Depends)
---
 debian/changelog |    6 ++++++
 dh_installdeb    |    2 ++
 2 files changed, 8 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 526ca9f..ffdc5b1 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,8 +1,14 @@
 debhelper (9.20131128) UNRELEASED; urgency=low
 
+  [ Joey Hess ]
   * cmake: Configure with -DCMAKE_BUILD_TYPE=None
     Closes: #701233
 
+  [ Andreas Beckmann ]
+  * dh_installdeb: Add support for dpkg-maintscript-helper commands
+    symlink_to_dir and dir_to_symlink that were added in dpkg 1.17.2.
+    Closes: #731723
+
  -- Joey Hess <joeyh@debian.org>  Thu, 05 Dec 2013 14:04:01 -0400
 
 debhelper (9.20131127) unstable; urgency=low
diff --git a/dh_installdeb b/dh_installdeb
index e233170..1f02edf 100755
--- a/dh_installdeb
+++ b/dh_installdeb
@@ -69,6 +69,8 @@ init();
 my %maintscript_predeps = (
 	"rm_conffile" => "",
 	"mv_conffile" => "",
+	"symlink_to_dir" => "1.17.5",
+	"dir_to_symlink" => "1.17.5",
 );
 
 foreach my $package (@{$dh{DOPACKAGES}}) {
-- 
1.7.10.4

